less/components/btn.less

```diff
@@ -1,4 +1,5 @@
 .btn {
+  white-space: normal;
```

Comment:
button text should be wrapped.  we shouldn't assume button text can always fit into one line, especially the site will be localized in the future.